recipes-extended/pigz/pigz_2.2.4.bbappend

```diff
@@ -1,2 +0,0 @@
-SRC_URI = "${DEBIAN_MIRROR}/main/p/pigz/pigz_${PV}.orig.tar.gz \
-           file://ldflags.patch"
```

Comment:
I don't think it is. Its not clear to me why it needed at all. It builds just find an works without either the modified source package or the patch.